JAVASCRIPT GAME/Jump Game/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en" onkeydown="myFunction(event)">
+<head>
+    <meta charset="UTF-8">
+    <title>Jump Game</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h2>Jump Game</h2>
+    <p id="co">Press space to jump</p>
+    <div class="game">
+        <div id="character"></div>
+        <div id="block"></div>
+        <div id="block2"></div>
+    </div>
+    <p>Score: <span id="scoreSpan"></span></p>
+</body>
+<script src="script.js"></script>
+</html>
```

Comment:
Add a new line at the end of file.

Use https://codebeautify.org/htmlviewer to beautify your HTML part.